roles/mcpserver-model/templates/model.deployment.yaml.j2

```diff
@@ -0,0 +1,215 @@
+{% endif %}
+      initContainers:
+{% if bundle_ca_crt or is_openshift %}
+      - name: configure-combined-ca-bundle
+        image: {{ _image }}
+        imagePullPolicy: '{{ image_pull_policy }}'
+        command:
+          - /bin/sh
+          - -c
+          - |
+            echo "SSL Manager: Creating combined CA bundle"
+            mkdir -p /etc/ssl/combined-ca/
+            mkdir -p /etc/pki/ca-trust/extracted/{java,pem,openssl,edk2}
+
+            # First extract the CA trust files so the system CAs are available
+            echo "SSL Manager: Extracting system CA trust files..."
+            update-ca-trust extract --output /etc/pki/ca-trust/extracted
+
+            # Now use the extracted system CAs from certifi/ca-trust
+            echo "SSL Manager: Adding system CAs from certifi..."
+            python3 -c "import certifi; print(certifi.where())" | xargs cat > /etc/ssl/combined-ca/ca-bundle.crt
+{% if is_openshift %}
+            # Add OpenShift service CA
+            if [ -f /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt ]; then
+              echo "# OpenShift Service CA" >> /etc/ssl/combined-ca/ca-bundle.crt
+              cat /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt >> /etc/ssl/combined-ca/ca-bundle.crt
+              echo "SSL Manager: Added OpenShift service CA"
+            fi
+{% endif %}
+{% if bundle_ca_crt %}
+            # Add customer CA bundle
+            if [ -f /etc/pki/ca-trust/source/anchors/bundle-ca.crt ]; then
+              echo "# Customer CA Bundle" >> /etc/ssl/combined-ca/ca-bundle.crt
+              cat /etc/pki/ca-trust/source/anchors/bundle-ca.crt >> /etc/ssl/combined-ca/ca-bundle.crt
+              echo "SSL Manager: Added customer CA bundle"
+            fi
+{% endif %}
+            chmod 644 /etc/ssl/combined-ca/ca-bundle.crt
+            echo "SSL Manager: Combined CA bundle created successfully"
+
+            # Verify the bundle contains both system and OpenShift CAs
+            echo "SSL Manager: CA bundle contains $(grep -c 'BEGIN CERTIFICATE' /etc/ssl/combined-ca/ca-bundle.crt) certificates"
+{% if combined_api.resource_requirements is defined %}
+        resources: {{ combined_api.resource_requirements }}
+{% endif %}
+        volumeMounts:
+{% if is_openshift %}
+          - name: server-tls-certs
+            mountPath: /var/run/secrets/kubernetes.io/serviceaccount/
+            readOnly: true
+{% endif %}
+{% if bundle_ca_crt or is_openshift %}
+          - name: combined-ca-bundle
+            mountPath: /etc/ssl/combined-ca/
+          - name: "ca-trust-extracted"
+            mountPath: "/etc/pki/ca-trust/extracted"
+{% endif %}
+{% if bundle_ca_crt %}
+          - name: "{{ ansible_operator_meta.name }}-bundle-cacert"
+            mountPath: /etc/pki/ca-trust/source/anchors/bundle-ca.crt
+            subPath: bundle-ca.crt
+            readOnly: true
+{% endif %}
+{% endif %}
+      containers:
+      - name: model-api
```

Comment:
Nit; you mind find reference to "model" misleading when this is a MCP Server.

I know you've copied/pasted this.